src/shared/useComputePositionId.ts

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState, useRef } from 'react';
+import {
+  Position,
+  PositionId,
+} from '@penumbra-zone/protobuf/penumbra/core/component/dex/v1/dex_pb';
+
+export function useComputePositionId() {
+  const ref = useRef<(position: Position) => PositionId>();
+  const [, setIsSet] = useState(false);
+
+  useEffect(() => {
+    const set = async () => {
+      // cant import directly without breaking the build cmd
```

Comment:
question: why does that happen I wonder. Is it because the wasm file is made for web and not nodejs?

Reply:
I don't know the reason tbh..